app/views.py

```diff
@@ -0,0 +1,30 @@
+from app import app
+from flask import render_template
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/<input>')
+def print_user_input(input):
+    if input.__contains__('='):
+        variable, value = input.split("=")
```

Comment:
Be consistent with your use of single-quotes and double-quotes. Choose one. Stick with it.